src/pathfinding_service/api.py

```diff
@@ -33,12 +34,14 @@
 from raiden_libs.types import Address, TokenNetworkAddress
 
 log = structlog.get_logger(__name__)
+# change maxlen in case more than 20 paths are requested
+last_requests: collections.deque = collections.deque([], maxlen=20)
```

Comment:
20 could be a bit small for large scenarios

Reply:
changed it to 200